src/errors/runtime/message.js

```diff
@@ -22,9 +22,9 @@ export default {
     testBodyIsNotAFunction:             'The test body is expected to be a function, but it was "{type}".',
     beforeEachIsNotAFunction:           `Fixture's "beforeEach" method takes a function, but "{type}" was passed.`,
     afterEachIsNotAFunction:            `Fixture's "afterEach" method takes a function, but "{type}" was passed.`,
-    clientCodeIsNotAFunction:           'Client code is expected to be specified as a function, but "{type}" was passed.',
+    hybridFunctionCodeIsNotAFunction:   'Hybrid function code is expected to be specified as a function, but "{type}" was passed.',
```

Comment:
Can we pass the function type here, so that it becomes

```
    hybridFunctionCodeIsNotAFunction:   '{functionType} function code is expected to be specified as a function, but "{type}" was passed.'
```